README.md

```diff
@@ -1,34 +1,74 @@
 
-The app was made using a TDD process and utilising an MVC model to access data from tables from a small database using SQL. 
+<p>The app was made using a TDD process and utilising an MVC model to access data from tables from a small database using SQL.</p>
 
-STEP BY STEP
-There are a few things you'll need to do if you want to get it up and running.
+---
 
-If you want to clone the repo I would recommend forking first incase you want to make any changes then using the command:
-git clone <YOUR_URL_HERE>  you will be able to copy the files to your own local repo.
+## STEP BY STEP
 
-DEPENDENCIES 
+<p>There are a few things you'll need to do if you want to get it up and running.</p>
 
-make sure to install all the dependencies first, this can be done using the command line prompt npm i to install all the appropriate dependencies. They should include: dotenv, express, postgres, supertest, jest, jest-sorted, jest-extended, husky and pg-format.
+<p>If you want to clone the repo I would recommend forking first incase you want to make any changes.<br>
+You will then want to copy the url for your new repo for use in cloning locally.</p>
 
-SETUP DATABASE
+<p>Then you will want to clone the repo to a local directory.
+first navigate to the folder you wish to copy the directory into and clone it using the git clone command in your terminal:
 
-You will need to make sure you connect the two databases (development and test).
-To do this you will need to create individual .env files using the .env-example as a template.
+`git clone <YOUR_URL_HERE> `
 
-to seed the databases you will need to run the command npm run setup-dbs, this will allow the app to run as intended. 
+once the files are cloned you will need to install the dependencies</p>
 
-MINIMUM REQUIREMENTS:
+### DEPENDENCIES
+
+<p>to install the dependencies I have used you will need to use the prompt:
+
+`npm i `
+
+They should include: dotenv, express, postgres, supertest, jest, jest-sorted, jest-extended, husky and pg-format. They should be in the package.json like this:</p>
+<code> "devDependencies": {<br>
+"husky": "^7.0.4",<br>
+"jest": "^27.5.1",<br>
+"jest-extended": "^2.0.0",<br>
+"jest-sorted": "^1.0.14",<br>
+"pg-format": "^1.0.4"<br>
+},<br>
+"dependencies": {<br>
+"dotenv": "^16.0.0",<br>
+"express": "^4.17.3",<br>
+"pg": "^8.7.3",<br>
+"supertest": "^6.2.2"<br>
+},</code>
+
+### SETUP DATABASE
+
+<p>You will need to make sure you connect the two databases (development and test).
+To do this you will need to create individual **.env** files using the **.env-example** as a template.<br>
+
+You will require a **.env.test** and a **.env.development** file to be able to connect to the appropriate database. </p>
+
+<p>to seed the databases you will need to run the command
+
+`npm run setup-dbs `
+
+this will allow the app to run as intended.</p>
+
+---
```

Comment:
don't forget about seeding the database

Reply:
Adding to what Jade said, after running setup-dbs you also want to run `npm run seed` and `npm run start` (hint: look at your scripts in your package.json file).